pages/common/shar.md

```diff
@@ -0,0 +1,28 @@
+# shar
+
+> Create a shell archive.
+> More information: <https://manned.org/shar>.
+
+- Specify compression for input files:
```

Comment:
Sorry, but this example doesn't work. No file is specified so the command expects a list of files from stdin. These are possible variants:
`shar --intermix-type file.sh` or `cat list.txt | shar --intermix-type`

Also the example description doesn't explain what the whole example does.